src/wm/title.rs

```diff
@@ -106,15 +106,30 @@ impl Title {
 
     pub fn from_title(title: &str, lang: &str) -> Result<Self, ParseTitleError> {
         let title = title.trim();
-        let lang = lang.trim();
         if title.is_empty() {
             return Err(ParseTitleError::NoTitle);
         }
+        // Wikipedia titles must be less than 256 bytes of UTF-8.
+        // See: https://en.wikipedia.org/wiki/Wikipedia:Naming_conventions_(technical_restrictions)#Title_length
+        if !title.len() < 256 {
+            return Err(ParseTitleError::TitleLong);
+        }
+
+        let lang = lang.trim();
         if lang.is_empty() {
             return Err(ParseTitleError::NoLang);
         }
+        // Longest Wiki name is 'zh-classical' (12 chars).
+        // See: https://meta.wikimedia.org/wiki/List_of_Wikipedias
+        if lang.len() > 12 {
```

Comment:
What if new, longer names will be added?

Reply:
I've removed that check, we can add warnings later if it is a problem.